src/jdk.jpackage/share/classes/jdk/jpackage/internal/AddLauncherArguments.java

```diff
@@ -120,6 +122,15 @@ private void initLauncherMap() {
         Arguments.putUnlessNull(bundleParams, CLIOptions.ICON.getId(),
                 (value == null) ? null : Path.of(value));
 
+        Arguments.putUnlessNull(bundleParams, SHORTCUT_HINT.getID(),
```

Comment:
I think it would be better to add platform-specific options only if jpackage runs on that platform:
```
if (Platform.isWindows())  {
    Arguments.putUnlessNull(bundleParams, SHORTCUT_HINT.getID(), getOptionValue(CLIOptions.WIN_SHORTCUT_HINT));
    Arguments.putUnlessNull(bundleParams, MENU_HINT.getID(), getOptionValue(CLIOptions.WIN_MENU_HINT));
}

if (Platform.isLinux())  {
    Arguments.putUnlessNull(bundleParams, SHORTCUT_HINT.getID(), getOptionValue(CLIOptions.LINUX_SHORTCUT_HINT));
}

```